docs/source/geometry.transform.rst

```diff
@@ -75,6 +76,14 @@ Module
 .. autoclass:: Rescale
 .. autoclass:: Affine
 .. autoclass:: HomographyWarper
-.. autoclass:: Homography
-.. autoclass:: Similarity
-.. autoclass:: ImageRegistrator
+
+
+Image registration
+------------------
+
+.. image:: _static/img/registration.gif
+  :width: 400
+  :alt: Image registration with ImageRegistrator module
+
+.. automodule:: kornia.geometry.transform.image_registrator
+    :members:
```

Comment:
I would also somehow create a new section in the main index here, under Get Started
https://github.com/kornia/kornia/blob/master/docs/source/index.rst

We can name it `Kornia Algorithms` (or similar) and explain about high level plans, etc. Besides, should we expose `ImageRegistrator` to top level access ?

Reply:
Could you move it then, please? I am not that comfortable with multi-level sphinx docs. 
Algorithms sounds algorithmic. Maybe "Applications"?

Reply:
sure I'll do